user_bzz_read_spec.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Swarm BZZ read spec
+about: Swarm BZZ read spec
+title: Swarm BZZ read spec
```

Comment:
Isn't BZZ light = BZZ read + BZZ readwrite?

Reply:
What's BZZ full in that case? BZZ light is more about (not) syncing as far as I understood it.